lib/pymedphys/_experimental/streamlit/apps/wlutz/_calculation.py

```diff
@@ -481,3 +480,10 @@ def _calculate_wlutz(
     )
 
     return field_centre, bb_centre
+
+
+def _filepath_to_filename(path):
```

Comment:
this is fully or partially qualified path to just the local filename (unqualified)?

Reply:
Yes, the only place this is used is within this same `_calculation.py` module to create a `filename` column within the dataframe by calling it as such:

```python
database_table["filename"] = database_table["filepath"].apply(_filepath_to_filename)
```